torch/utils/model_zoo.py

```diff
@@ -6,12 +6,21 @@
 import shutil
 import sys
 import tempfile
-if sys.version_info[0] == 2:
-    from urlparse import urlparse
-    from urllib2 import urlopen
-else:
-    from urllib.request import urlopen
-    from urllib.parse import urlparse
+
+try:
+    from requests.utils import urlparse
+    import requests.get as urlopen
+    requests_available = True
+except ImportError:
+    requests_available = False
+
+if not requests_available:
```

Comment:
nit: you can include that as part of `except` and remove the `if`

Reply:
yeah - had it this way first, just dind't like the multiple indentations. Happy to change back to this if you prefer?

Reply:
Oh the nit is because you dont need to change the indent. It’s enough to remove these two lines. But I don’t care, it’s fine either way :)

Reply:
Ah yeah, have changed this now